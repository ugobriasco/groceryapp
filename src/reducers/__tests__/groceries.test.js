import reducer from '../groceries';


// opt: no snapshots usage
// it('sets initial state', () => {
// 	const expected={
// 		groceriesData: {list: []},
// 		groceriesView: [],
// 		error: null
// 	};
// 	const actual = reducer(undefined, {});
// 	expect(actual).toEqual(expected)

// });

it('sets initial state', () => {
	expect(reducer(undefined,{})).toMatchSnapshot();
});



