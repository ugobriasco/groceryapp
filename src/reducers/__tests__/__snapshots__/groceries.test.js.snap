// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sets initial state 1`] = `
Object {
  "error": null,
  "groceriesData": Object {
    "list": Array [],
  },
  "groceriesView": Array [],
}
`;
